src/System.Net.Primitives/tests/FunctionalTests/CookieCollectionTest.cs

```diff
@@ -121,6 +121,8 @@ public static void IndexSubscript_Get_Success()
             Assert.Equal(cc["name1"], c1);
             Assert.Equal(cc["name2"], c2);
             Assert.Equal(cc["name3"], c4);
+
+            Assert.Null(cc["no such name"]);
```

Comment:
I don't understand this change. The "Success" in the name means that you are able to extract the cookie. This line asserts that you cannot.
Please undo this change from this and below test.